js/main.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
+
+var baseSelect = document.querySelector("#base");
+var symbolSelect = document.querySelector("#symbol");
+var display = document.querySelector(".display-block");
+
+function start() {
+  var xhr = new XMLHttpRequest();
+  xhr.open("GET", "https://api.ratesapi.io/api/latest");
+  xhr.send();
+  xhr.onload = function () {
+    if (xhr.status !== 200) {
+      throw new Error("Ошибка " + xhr.status + ": " + xhr.statusText);
+    } else {
+      var data = JSON.parse(xhr.response);
+      baseSelect.append(new Option("EUR", "EUR"));
+      symbolSelect.append(new Option("All", ""));
+
+      for (var currency in data.rates) {
+        baseSelect.append(new Option(currency, currency));
+        symbolSelect.append(new Option(currency, currency));
+      }
+    }
+  };
+}
+
+function getCurrency() {
+  var xhr = new XMLHttpRequest();
+  xhr.open(
+    "GET",
```

Comment:
more preferable is to declare variable above like `"https://api.ratesapi.io/api/latest?base=" + baseSelect.value + "&symbols=" + symbolSelect.value` and use here. Also you can use ES6 and above